### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by Keats**

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -28,10 +27,19 @@ import { render } from 'react-dom'
 import { createStore, combineReducers } from 'redux';
 
 // 1. Connect your data, similar to react-redux @connect
+// Decorator version
 @asyncConnect({
-  lunch: (params, helpers) => Promise.resolve({id: 1, name: 'Borsch'})
+  lunch: (params, store, helpers) => Promise.resolve({id: 1, name: 'Borsch'})
```
Comment (sars) [inline README.MD:33]: there is no store in the second arg here...
It's ilocated in helpers.store

File: README.MD [hunk 2]
```diff
@@ -28,10 +27,19 @@ import { render } from 'react-dom'
...
 })
 class App extends React.Component {
+  // 1. Same as the decorator but as a function
+  static reduxAsyncConnect(params, store, helpers) {
```
Comment (sars) [inline README.MD:47]: you should use decorator  OR static method. decorator in tour example will override this method

File: README.MD [hunk 3]
```diff
@@ -28,10 +27,19 @@ import { render } from 'react-dom'
...
+    const {dispatch, getState} = store;
+    if (!helpers.isLoaded(getState())) {
+      return Promise.all([dispatch(loadWidgets())]);
```
Comment (sars) [inline README.MD]: there is no loadWidgets defined...

may be it's better here to add just some setTimeout?
this method is just for delay rendering - you can do here whatever you want. dispatch actions in particular

**Comment by sars** [comment]
@Keats , thanks, it's great job
I'm not strong in english, but do my best :)

File: README.MD (hunk 1 above)
Comment (Keats) [inline README.MD:33]: ah i thought it was a mistake. How come the api is different for the decorator and the static method?
Actually i'm not sure i see the point of return a value to the decorator, if you're going to to use redux, the data will be fetched by `connect` automatically when the reducers get updated

File: README.MD (hunk 2 above)
Comment (Keats) [inline README.MD:47]: it was to keep it as one example, i'll split it off

File: README.MD (hunk 3 above)
Comment (Keats) [inline README.MD]: i'll clarify the examples (except the one on the server as i don't use node there myself so have no idea what it's supposed to be)

File: README.MD (hunk 1 above)
Comment (Keats) [inline README.MD:33]: @sars any comment on that?

**Comment by Keats** [comment]
Updated the examples minus the decorator question above.
Also relaxed the peer deps as it currently cannot be installed with the latest version of react-redux

File: README.MD (hunk 1 above)
Comment (sars) [inline README.MD:33]: @Keats 

> How come the api is different for the decorator and the static method?

Yeah... it's my fault....
i actually thought to change interface significantly:
https://github.com/Rezonans/redux-async-connect/issues/16#issuecomment-178227211
what do you think?

**Pull request closed without merging**